config/containers/runmetrics.md

```diff
@@ -64,6 +76,41 @@ the hierarchy mountpoint. `/` means the process has not been assigned to a
 group, while `/lxc/pumpkin` indicates that the process is a member of a
 container named `pumpkin`.
 
+#### cgroup v2
+
+On cgroup v2 hosts, the content of `/proc/cgroups` isn't meaningful.
+See `/sys/fs/cgroup/cgroup.controllers` to the available controllers.
+
+### Changing cgroup version
+
+Changing cgroup version requires rebooting the entire system.
+
+On systemd-based systems, cgroup v2 can be enabled by adding `systemd.unified_cgroup_hierarchy=1`
+to the kernel cmdline.
+To revert the cgroup version to v1, you need to set `systemd.unified_cgroup_hierarchy=0` instead.
+
+If `grubby` command is available on your system (e.g. on Fedora), the cmdline can be modified as follows:
+
+```console
+$ sudo grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=1"
+```
+
+If `grubby` command is not available, edit the `GRUB_CMDLINE_LINUX` line in `/etc/default/grub`
+and run `sudo update-grub`.
+
+### Running Docker on cgroup v2
+
+Docker supports cgroup v2 experimentally since Docker 20.03.
+Running Docker on cgroup v2 also requires the following conditions to be satisfied:
+* containerd: v1.4 or later
+* runc: v1.0.0-rc91 or later
+* Kernel: v4.15 or later (v5.2 or later is recommended)
+
+Note that the cgroup v2 mode behaves slightly different from the cgroup v1 mode:
+* The default cgroup driver (`dockerd --exec-opt native.cgroupdriver`) is "systemd" on v2, "cgroupfs" on v1.
+* The default cgroup namespace mode (`docker run --cgroupns`) is "private" on v2, "host" on v1.
```

Comment:
This is not merged yet, right?

Reply:
merged: https://github.com/moby/moby/pull/40174/commits/19baeaca267d5710907ac1b3c3972d44725fe8ad

An unmerged one is only for old API clients: https://github.com/moby/moby/pull/41072